webservices/legal_docs/advisory_opinions.py

```diff
@@ -182,12 +185,14 @@ def get_documents(ao_id, bucket):
                 "text": row["ocrtext"],
                 "date": row["document_date"],
             }
-            pdf_key = "legal/aos/%s.pdf" % row["document_id"]
-            logger.info("S3: Uploading {} Orig filename: {}".format(pdf_key, document['filename']))
+            pdf_key = "legal/aos/{0}/{1}".format(row['ao_no'],
+                    str.replace(row["filename"] or '', ' ', '-'))
```

Comment:
As discussed offline, if the `or` is only to handle missing filenames in test data, we can fix the test data.